src/bootstrap/src/core/build_steps/doc.rs

```diff
@@ -1134,7 +1145,8 @@ tool_doc!(
         "crates-io",
         "mdman",
         "rustfix",
-    ]
+    ],
+    allow_features: "specialization"
```

Comment:
Suggestion: can you please add a reason for the _why_? That this is because

- `im-rc` is a dependency
- It does auto opt-in to `feature(specialization)` in its build script based only on the channel being nightly

Reply:
Done :+1: